src/controllers/userController.js

```diff
@@ -0,0 +1,76 @@
+const userService = require('../services/userService');
+
+const get = (req, res) => {
+  res.send(userService.getAllUsers());
+};
+
+const getOne = (req, res) => {
+  const { id } = req.params;
+
+  const user = userService.getUserById(id);
+
+  if (!user) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(user);
+};
+
+const update = (req, res) => {
+  const { id } = req.params;
+  const { name } = req.body;
+  const user = userService.updateUser({ id, name });
+
+  if (!user) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  if (typeof name !== 'string') {
+    res.sendStatus(422);
+
+    return;
+  }
+
+  res.send(user);
+};
+
+const create = (req, res) => {
+  const { name } = req.body;
+
+  if (!name) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const user = userService.createUser(name);
+
+  res.statusCode = 201;
+  res.send(user);
+};
+
+const remove = (req, res) => {
+  const { id } = req.params;
+
+  if (!userService.getUserById(id)) {
```

Comment:
In the 'remove' function, you're fetching the user from the database to check if they exist before deleting them. This results in two database calls which is not performant. You could simply attempt to delete the user and handle the case where the user doesn't exist based on the response from the delete operation. This is a performance issue.